app/models/ticket.rb

```diff
@@ -74,6 +74,6 @@ def self.group_tickets(period)
   end
 
   def self.user_tickets(user)
-    Ticket.joins(:parking_zone).where(parking_zones: {user: user}) #.where(entry: Time.zone.today)
+    Ticket.joins(:parking_zone).where(parking_zones: {user: user}).where(entry: Time.zone.now.beginning_of_day..Time.zone.now.end_of_day)
```

Comment:
where(exit.....)

Reply:
No se supone que todo debe estar cerrado del día anterior?

Reply:
Pero el arqueo es para ver en medio de un día cuanta plata tiene que tener, no?

Reply:
@dlopezg3 Está bien... si decís que así está bien i'll trust you!